packages/core/src/api/browser.ts

```diff
@@ -23,6 +23,7 @@ export interface ConnectorSDK extends CallbackManager {
   reconnect(connectorId: string, options: ConnectorSDKReconnectOptions): ConnectorSDKConnector
 
   reset(): void // For Test and Development Use
+  resetConnector: (connectorId: string) => void // For forcing a reset of a specific connector
```

Comment:
how is this different from `reset()` we already have? could we just leverage that? i'm not a fan of the name of this method